src/main/java/net/sf/jabref/model/DuplicateCheck.java

```diff
@@ -16,25 +18,28 @@
 import net.sf.jabref.model.entry.FieldProperty;
 import net.sf.jabref.model.entry.InternalBibtexFields;
 
+import info.debatty.java.stringsimilarity.Levenshtein;
```

Comment:
Yes, from that perspective it is OK for sure and we can of course revive the old code if required later. Oh, here I found it: https://github.com/koppor/jabref/issues/131